src/assets/crisistextline/sass/partials/03-components/_main.scss

```diff
@@ -13,7 +13,9 @@
             width: 40em;
             height: auto;
             max-height: 100%;
-            margin: 5em auto;
+            margin-top: 5em;
+            @include calc(margin-left, 50% - #{20em}/2 - #{$width-sidebar-left});
+            @include calc(margin-right, 50% - #{20em}/2);
```

Comment:
what's this 20em accounting for? would it make sense to move it into a new variable?